src/Appwrite/Utopia/Response/Model/Bucket.php

```diff
@@ -17,16 +17,16 @@ public function __construct()
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('$createdAt', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'Bucket creation date in Unix timestamp.',
-                'default' => 0,
-                'example' => 1592981250,
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Bucket creation date in Datetime',
```

Comment:
Let's say `creation time` instead of `creation date`. It's not just a date, but also includes time of day.